### PR TITLE
Fix `packages.vscode-inferno-syntax-highlighting`

### LINKED ISSUE #15: `vscode-inferno-syntax-highlighting` is broken
#11 removed the `vsce` input from the packages defined in `vsCodeInfernoFor`. This needs to be added back for `vscode-inferno-syntax-highlighting`:

```
error: builder for '/nix/store/...-vscode-inferno-syntax-highlighting-0.0.1.drv' failed with exit code 127;
       last 10 log lines:
       > patching sources
       > updateAutotoolsGnuConfigScriptsPhase
       > configuring
       > patching script interpreter paths in .
       > building
       >
       > > vscode-inferno-syntax-highlighting@0.0.1 build-tm
       > > node build-tm
       >
       > /nix/store/ncm43bbk3nfwpf7pycgz1cnasnihcmkg-stdenv-darwin/setup: line 1408: vsce: command not found
```

### PR DESCRIPTION
**Pull request by ngua**
Closes #15

The package's `npmBuild` calls `vsce`, but this was removed from the `buildInputs` in a previous PR. The `@vscode/vsce` dependency was also missing from `package.json`

### CODE REVIEW

File: vscode-inferno-syntax-highlighting/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 PanAeon
```
Comment (mirko-plowtech) [inline vscode-inferno-syntax-highlighting/LICENSE:3]: ???

File: vscode-inferno-syntax-highlighting/package.json [hunk 2]
```diff
@@ -12,23 +12,24 @@
...
+        "inferno",
+        "lsp",
+        "language server"
+    ],
     "repository": {
-		"type": "git",
-		"url": "https://github.com/plow-technologies/all.git"
-	},
+        "type": "git",
+        "url": "https://github.com/plow-technologies/all.git"
```
Comment (mirko-plowtech) [inline vscode-inferno-syntax-highlighting/package.json:31]: wrong url? (not related to the PR, but just in case)

**Review by mirko-plowtech — APPROVED** [review]
lgtm! (just minors)

File: vscode-inferno-syntax-highlighting/LICENSE (hunk 1 above)
Comment (ngua) [reply vscode-inferno-syntax-highlighting/LICENSE:3]: Copied from the existing license file in `vscode-inferno-lsp-server` (there was no license before). I also found the attribution strange, I'll change both of them

File: vscode-inferno-syntax-highlighting/package.json (hunk 2 above)
Comment (ngua) [reply vscode-inferno-syntax-highlighting/package.json:31]: Ah yes, this must predate the `all`-`inferno` split

Comment (ngua) [reply vscode-inferno-syntax-highlighting/package.json:31]: I'll also change this

**Pull request merged**